File: lessons/compiled_lang/java/src/j20.arguments.java
// Not public - see a00.output.java for why.
class J20Arguments {
    public static void main(String[] args) {
        System.out.println("The arguments passed are (reverse order):");
        for (int i = args.length - 1; i >= 0; i--) {
            System.out.println(" item " + (i + 1) + ": " + args[i]);
        }
    }
}

File: lessons/compiled_lang/cs/src/j20.arguments.cs
using System;

class J20Arguments
{
    static void Main(string[] args)
    {
        Console.WriteLine("The arguments passed are (reverse order):");
        for (int i = args.Length - 1; i >= 0; i--)
        {
            Console.WriteLine(" item " + (i + 1) + ": " + args[i]);
        }
    }
}
